Bottom: 69cbf36d9d1ec7043826903ae52decc182c4c9d5
Top:    6c67cf725223f0698727161b9c3dddc94f5f36b4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-06-09 12:35:54 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 9c6d8ee..d52515f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,6 +1238,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1260,6 +1261,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
